### ISSUE TITLE
Freezing/no progress then Crash in benchmark 

### ISSUE THREAD

**Issue by McMcNugget** [nicehash/NiceHashMiner-Archived#884]
well as the title states , NHM just does not wanna benchmark , it runs fine on my main system , but just fails and crashes on any other install , this is w/ a fresh windows 7 install , up to date and all drivers installed 
the GPU's in Question are : 
Radeon HD 5850
Radeon HD 6870
Radeon R9 270X 

Driver used is 15.7.1

_Yes i am aware these are **NOT** profitable and so on , but i **dont** pay for power (fixed rate no matter the consumption) so i might aswell make another 50$ a month and keep my room warm at the same time_

does anyone have benchmark .json files to bypass it? 
the miner runs fine when launched standalone , it just does not make me any money as its not configured for my wallet and all

**Comment by nicehashdev** [comment]
It is not mandatory to run benchmarks. You can configure speeds manually in settings.

**Comment by McMcNugget** [comment]
could you please tell me how? 
that would help alot , thank you

**Comment by undefy** [comment]
Does the bench mark crash straight away or at certain algo? I had similar problem on my gtx (not amd) for excavator algo.

1.During bench mark i wait for number to show up in the last column. 
2. Press stop. This successfully completes benchmark. If i do not stop manually the screen goes black and i have to redo everything from 0.
3. Hard reset the PC. 
4. Boot up. Repeat on next algo until all algos are sorted.

**Comment by McMcNugget** [comment]
the benchmark does not show any progress just "..." for the entire time its set to in settings (defaults btw) and then NHM crashes completely

**Comment by undefy** [comment]
Yup, look over to the right . It goes ..., some number, blank. Does a number appear after 2-4 minutes where the blank column is? i.e. ..., some number, some number. If the 3rd column gets a number stop the bench mark and do the procedure i told you about.

**Comment by McMcNugget** [comment]
nope , nothing happens , after 2 or so minutes (the stock benchmark time on standard settings) it just chrashes
anyways , i figured out how to override benchmark numbers in the settings , and how im facing a differend issue , im getting a 100% reject rate on my 6870 in Cryptonight and decred , both times its "share above target"  (see attachment)
![capture55555](https://cloud.githubusercontent.com/assets/28737741/26222211/7514b5b4-3c19-11e7-920b-d7484651b9a9.PNG)
